public/javascripts/SVLabel/src/SVLabel/Storage.js

```diff
@@ -39,6 +39,10 @@ function TemporaryStorage(JSON, params) {
             set("completedOnboarding", null);
         }
 
+        if (!get("completedMissionAnonymously")){
```

Comment:
Hmm, so if we use `completedFirstMission` for both types of users, then it should be updated to `completedFirstMission` to cover both users.